Makefile-test.am

```diff
@@ -99,6 +104,10 @@ endif #UNIT
 
 if ENABLE_INTEGRATION
 noinst_LTLIBRARIES += test/integration/libtest_utils.la
+
+if TESTPTPM
```

Comment:
instead of doing if else endif just do:
if !TESTTPM
endif

Reply:
Yes that's true I meant 12.2.3.5 TPMS_RSA_PARMS. Sorry.